jest.config.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   roots: ["<rootDir>/src/", "<rootDir>/test/"],
   moduleFileExtensions: ["ts", "tsx", "js", "jsx"],
   transform: {
-    "^.+\\.(js|jsx|ts|tsx)$": "babel-jest",
+    //"^.+\\.(ts|tsx)$": "babel-jest",
```

Comment:
This is ok when developing, but commented-out lines shouldn't be committed.

Reply:
True. During Testing it was quite helpful, since this is the best place to change it back and forth between the old transformer and the new.